config/locales/ar.yml

```diff
@@ -212,6 +227,18 @@ ar:
         other: مسودة النصوص
         two:
         zero:
+      edition:
```

Comment:
I think the pluralisation needs updating here, to reflect the plurals used in each language (e.g. Arabic has `one`, `many`, `few`, `other`, `two` and `zero` that will need placeholder keys).

Reply:
Ah yes you're right, good spot! I have added / removed pluralisations for the other apps but Whitehall must've slipped through the net.